api/chat.go

```diff
@@ -280,12 +280,20 @@ func (r chatRoutes) handleApprove(ctx tools.TUMLiveContext, msg []byte) {
 	if ctx.User == nil || !ctx.User.IsAdminOfCourse(*ctx.Course) {
 		return
 	}
+
 	err = r.ChatDao.ApproveChat(req.Id)
 	if err != nil {
 		log.WithError(err).Error("could not approve chat")
+		return
+	}
+
+	chat, err := r.ChatDao.GetChat(req.Id, 0)
```

Comment:
I am not too familiar with the chat. Maybe this could use a comment to explain why the `userID` has to be `0` or, alternatively, a constant with a descriptive name.

Reply:
This actually quite interesting. @joschahenningsen do you have suggestion here, because we use the userId for `normalizeChat`. Its needed to set a flag `Liked` on the chat message (not sure why this is not handled on frontend). At our particular case the messages are approved so users have not liked it anyways, that means it should work with just settings the userId to 0, but its kind of ugly. Any good ideas to make it cleaner?

Reply:
Adding a comment that explains what these values do and why a zero is fine to use here would be nice, looks good otherwise.